### ISSUE TITLE
Model Selection for Baseline and Baseline++ Training with Base Classes

### ISSUE THREAD

**Issue by tao0420** [wyharveychen/CloserLookFewShot#34]
Hi there,

If I have the right understanding that for the baseline and baseline++ model trained with base classes, in the paper reported results the model used is the epoch399 if trained with miniImagenet?

I am wondering that for all the model structures, using the same epoch seems fair to compare in the paper however is there overfitting problem which may influence the evaluation performance as the model selection is not based on validation set for the base classes?

Thanks!

**Comment by wyharveychen** [comment]
Yes, for the 5-shot task, the model used is the epoch 399 if trained with mini-Imagenet.
For the 1-shot task, the model used is the epoch 599.

For the mini-Imagenet dataset, I do not observer the overfitting problem.
However, for the CUB dataset, the overfitting problem exists, and you can refer to issue #31 for a validation strategy.
